uber/templates/preregistration/preregbase.html

```diff
@@ -36,6 +36,7 @@
                 description: 'A single registration; you can register more before paying.',
                 onClick: function () {
                     $('.group_fields').addClass('hide');
+                    $('.non_group_fields').removeClass('hide');
```

Comment:
Here we are toggling the visibility of the "Promo Code" field when the user clicks between the "Attendee" and "Group" buttons on the preregistration page.